lib/net/tns/client.rb

```diff
@@ -58,6 +58,8 @@ def self.get_status(opts={})
             # a data length that is greater than the available data.
           end
 
+          response = conn.send_and_receive(ResendPacket.new())
```

Comment:
I would keep that part as is in this change since it works fine against servers I tested (9, 11.2).